java/src/main/native/src/TableJni.cpp

```diff
@@ -1533,8 +1533,7 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_readParquet(
 
     auto builder = cudf::io::parquet_reader_options::builder(source);
     if (n_filter_col_names.size() > 0) {
-      builder = builder.columns(n_filter_col_names.as_cpp_vector())
-                    .convert_binary_to_strings(n_col_binary_read.to_vector<bool>());
```

Comment:
I'm not sure this is right. We should be calling `set_column_schema()` here, shouldn't we?

Reply:
Yes, but that depends on changes to the plugin to build the schema and I was cutting this PR off here.

Reply:
Ok. We can do the final bit when the plugin is ready.